src/Tests/Microsoft.NET.TestFramework/ToolsetInfo.cs

```diff
@@ -17,10 +17,10 @@ namespace Microsoft.NET.TestFramework
 {
     public class ToolsetInfo
     {
-        public const string CurrentTargetFramework = "net7.0";
```

Comment:
There should be a better way instead of hard coding here and requiring modifying for each release. Who owns this?

Reply:
Why are you updating the currentTargetFramework for the tests in this PR?  That value affects what our tests target and also the NextTargetFramework should also probably stay as 8.0 for now as that was put into place to help transition between 7 and 8.  Our tests previously were all hardcoded individually and this was added to allow us to control most of them at once. I really don't want an automatic upgrade of this as it'll affect every test.

Reply:
Okay. I saw failures related to this version being `net7.0 ` as target framework not found and looking at the code, i felt it should be updated. Tests are still failing with same error.  Do you want me to revert this change?

Reply:
C:\h\w\A752091D\t\dotnetSdkTests\tklqedaq.ozp\It_errors_whe---BFA33BDB\MissingTargetPlatform\MissingTargetPlatform.csproj : error NU1012: Platform version is not present for one or more target frameworks, even though they have specified a platform: net8.0-custom
        Failed to restore C:\h\w\A752091D\t\dotnetSdkTests\tklqedaq.ozp\It_errors_whe---BFA33BDB\MissingTargetPlatform\MissingTargetPlatform.csproj (in 100 ms).
      StdErr:
      
      
      Stack Trace:
           at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
           at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
           at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
           at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
        /_/src/Tests/Microsoft.NET.TestFramework/Assertions/CommandResultAssertions.cs(61,0): at Microsoft.NET.TestFramework.Assertions.CommandResultAssertions.HaveStdOutContaining(String pattern)
        /_/src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAWindowsDesktopProject.cs(40,0): at Microsoft.NET.Build.Tests.GivenThatWeWantToBuildAWindowsDesktopProject.It_errors_when_missing_windows_target_platform(String propertyName)
           at InvokeStub_GivenThatWeWantToBuildAWindowsDesktopProject.It_errors_when_missing_windows_target_platform(Object, Object, IntPtr*)
           at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)

Reply:
The plan was to wait to update this value until after all of the templates had been updated to target 8.0. A bunch of project types are still defaulting to net7.0: https://github.com/dotnet/installer/blob/main/test/EndToEnd/ProjectBuildTests.cs#L399

I'd revert this for now and then fix the individual test to use the NextTargetFramework (that's why that was added).  Then we can do a separate PR to update the CurrentTF including any test fixes required for it.

Reply:
FYI you can use @dotnet/domestic-cat to reach our on-duty build engineer but it's @nagilson this week.